Update connect-to-tidb-cluster.md (#20)
tidbcloud/dbaas-docs: connect-to-tidb-cluster.md

# Connect to Your TiDB Cluster

> After your TiDB cluster is created on TiDB Cloud, you can use any of the following three methods to connect to your TiDB cluster:

Reviewer: I think we should provide a short explanation for these 3 connection methods. With the new feature landed, we have 2 connection methods both using the MySQL client, so users might want to know the difference.

Reviewer 2: And then know how to choose from the two. @sivagao PTAL

Author: You can access your cluster through SQL client, or quickly through online WebShell; according to your network environment, when using SQL client to connect, you can choose from standard connection which uses public ip and VPC peering.

Reviewer 2: I added a commit to update the description.

- [Connect via Standard Connection](#connect-via-standard-connection): to connect to the TiDB cluster by a SQL client via standard connection
- [Connect via VPC peering](#connect-via-vpc-peering): to connect to the TiDB cluster by a SQL client via VPC Peering
- [Connect via SQL shell](#connect-via-sql-shell): to try TiDB SQL and test out TiDB’s compatibility with MySQL quickly, or administer user privileges

## Connect via Standard Connection

To connect to your TiDB cluster via standard connection, perform the following steps:

1. Navigate to the **TiDB Cluster** page and find your newly created cluster.

2. Click **Connect**. The **Connect to TiDB** dialog displays.

3. Create the traffic filter for the cluster. Traffic filter is a list of IPs and CIDR addresses that are allowed to access TiDB Cloud via SQL client.

> If the cluster’s traffic filter is empty, follow the steps to add one.

Reviewer 2: Please see whether this is what you mean.
```suggestion
    - If it is the first time that you connect to the cluster, the traffic filter is empty by default. Perform the following su-steps to add one.
    - If the traffic filter is already set, skip this step.
```

    1. Click one of the buttons to add some rules quickly.

> - **Add Your Current IP Address**

Reviewer: These belongs to the first step, so should be indented.

> - **Add Your Current IP Address**
> - **Add Rules from Default Set**
> - **Allow Access from Anywhere**

Reviewer 2: Please keep the style and rules consistent with existing files. Otherwise, display errors might occur on the PingCAP website.
```suggestion
        - **Add Your Current IP Address**
        - **Add Rules from Default Set**
        - **Allow Access from Anywhere**
```

    2. Provide an optional description for the newly added IP address or CIDR range.
    

>     3. Click **Create Filter** button to confirm the changes.

Reviewer 2: ```suggestion
    3. Click **Create Filter** to confirm the changes.
```

4. Use a SQL client to connect to TiDB. 

    {{< copyable "shell" >}}

    ```shell
    mysql -u root -h <endpoint> -P 4000 -p
    ```

    > **Note:**
    >
    > Currently, TiDB does not work well with MySQL client 8.0, so you need to use MySQL client 5.7. Or you can add the `--default-auth=mysql_native_password --default-character-set` command line option if you are using MySQL client 8.0.

## Connect via VPC Peering

To connect to your TiDB cluster via VPC peering, perform the following steps:

1. Navigate to the **TiDB Cluster** page and find your newly created cluster.

2. Click **Connect**, and select the **VPC Peering** tab at the **Connect to TiDB** dialog.

3. Set up VPC peering. See [Set up VPC Peering](set-up-vpc-peering-connections.md) for details.

> 4. Click the Get Endpoint Button and wait for a few minutes. the Connect Command will display in the dialog

Reviewer: ```suggestion
4. Click the **Get Endpoint Button** and wait for a few minutes. The connection command will be displayed in the dialog.
```

> 4. Click the **Get Endpoint Button** and wait for a few minutes. The connection command will be displayed in the dialog.

Reviewer 2: ```suggestion
4. Click **Get Endpoint** and wait for a few minutes. Then the connection command displays in the dialog.
```

1. Navigate to the **TiDB Cluster** page and find your newly created cluster.

> 2. Click **Connect**, select the **Web SQL Shell** tab at the **Connect to TiDB** dialog, and click **Open SQL Shell**.
> 
> 3. On the prompted **TiDB password** line, enter the root password of the current cluster. Then your application is connected to the TiDB cluster.

Reviewer 2: ```suggestion
2. Click **Connect**, and select the **Web SQL Shell** tab at the **Connect to TiDB** dialog.

3. Click **Open SQL Shell**.

4. On the prompted **TiDB password** line, enter the root password of the current cluster. Then your application is connected to the TiDB cluster.
```
